python/lsst/verify/bin/dispatchverify.py

```diff
@@ -203,6 +204,7 @@ def build_argparser():
 def main():
     """Entrypoint for the ``dispatch_verify.py`` command line executable.
     """
+    logging.basicConfig(level=logging.INFO, stream=sys.stdout)
```

Comment:
You might want to consider playing with the log format as well by adding something like:

```python
format="{name} {levelname}: {message}", style="{")
```
to the end of the `basicConfig` call.

Reply:
I think Tim's suggestion is a good one, but you are the one who will have to read the outputs, so I leave it up to you.